### DISCUSSION TITLE
TokyuRuby会議15の記事を書きたい

### DISCUSSION THREAD

**Discussion by shokola** [rubima/magazine.rubyist.net#498] [記事を書きたい]
こんにちは、現在 2024年9月29日(日)開催のTokyuRuby会議15を計画しております。
その際の様子を記事にしようと思っています。

過去の記事は以下のような感じです。ちなみに14回の記事は担当者がアルコール摂取のためゲフンゲフン・・という感じで見送りになったようです。
https://github.com/rubima/magazine.rubyist.net/pull/218

自分は10年前・・？にRubyKaigiのレポートを書いた以来で、勝手など変わっていてご迷惑をかけるかもしれませんがよろしくお願いします🙇🙇🙇

**Comment by neko314** [comment]
ありがとうございます！ぜひお願いします👍

> 2024年9月29日(日)開催

ということを踏まえると、現在進行中の0064号ではなく、次かその次の号の記事として書いていただくことになりそう。
記事の作り方については、詳細は[doc/writing_process.md](https://github.com/rubima/magazine.rubyist.net/blob/0cf1aad5f6756edae182b6a57e942d75e8388eae/doc/writing_process.md)に任せるとして、ざっくりいうとPRを作っていただいて、こちらがmasterにマージして記事の公開となります。従来はその号のすべての記事をそろえて一斉に刊行していましたが、現在は個別にリリースし、ある区切りで号として完成させる流れになっています。

記事を書いていただくのは10月に入ってからになろうかと思いますが、ご不明な点があればいつでもお尋ねください〜

**Comment by shokola** [comment]
了解しました！なるほど、だから個別に記事のツイートしているのを見るのか！

> 記事を書いていただくのは10月に入ってから

はい、そんな予定です。疑問があればPRかここで質問させていただきますね🙇

**Reply by kakutani** [reply]
> 了解しました！なるほど、だから個別に記事のツイートしているのを見るのか！

「1個流し作戦」作戦は実験扱いで(see also #434 )、0063号はそのかたちで進めたのですが、わたしはリリース前に脱落してしまったのでした。この方式はいまも実験中だと思うので、気になる点などはフィードバックしてもらえるといいなと思ってます。

**Reply by shokola** [reply]
なるほど。

読者として個人的な意見ですが、
大抵自分は仕事の合間にツイートで流れてきて知って読むのですが(結構そんな人が多いのではないかと)、これまでは毎号ボリュームがあり、合間と言うこともあり、1〜3記事だけ読んで終わったりするので「1個流し作戦」のが丁寧に読めて良いなと思います！
